multiterm/src/Makefile.am

```diff
@@ -24,6 +24,8 @@ multiterm_la_SOURCES = \
 	terminal.vala
 
 multiterm_la_CFLAGS = \
+	-DGETTEXT_PACKAGE='"$(GETTEXT_PACKAGE)"' \
+	-DLOCALEDIR='"$(localedir)"' \
```

Comment:
> If you feel like doing it go ahead, otherwise this way is fine too, at least the globals are constant.

I won't do it today, but you can ;)

> Would it make more sense to have these in the Geany .vapi?

I don't think so no, because Geany doesn't provide it, nor depend on it.  Well, `GETTEXT_PACKAGE` a little as if it's not defined we will alias `_` to a no-op, but that's about it.